### PR TITLE
Update index.php

### PR DESCRIPTION
**Pull request by Papilla-Hitlers**

### CODE REVIEW

File: web/api/index.php [hunk 1]
```diff
@@ -34,6 +34,7 @@
 
     // Build query
     $cmdquery = VESTA_CMD.$cmd." ";
+    ksort($_POST);
     foreach($_POST as $k => $v) {
```
Comment (SysVoid) [inline web/api/index.php]: Please follow our conventions of `foreach (`

File: web/api/index.php [hunk 2]
```diff
@@ -34,6 +34,7 @@
...
         if(preg_match('/arg\d+/',$k)) {
```
Comment (SysVoid) [inline web/api/index.php]: Please follow our conventions of `if (`

File: web/api/index.php [hunk 3]
```diff
@@ -34,6 +34,7 @@
...
 		    $cmdquery = $cmdquery.escapeshellarg($v)." "; }
```
Comment (SysVoid) [inline web/api/index.php]: Syntax error. Please test your code before making any changes that could have quite a large impact.

**Comment by anton-reutov** [comment]
@Papilla-Hitlers write more about your fix

**Pull request closed without merging**